.github/PULL_REQUEST_TEMPLATE/base_template.md

```diff
@@ -0,0 +1,64 @@
+---
+name: Pull request template
+about: Template for interchain security templates
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+
+# Description
+
+Please include a summary of the changes and the related issue. If the the issue was ambiguous try to clarify it in this section.
+
+
+# Linked issues
+
+Closes: `#<issue>`
+
+
+# Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Non-breaking changes
+- [ ] New feature (non-breaking changes adding functionality)
+- [ ] Breaking change (fix, feature or refactoring that changes existing functionality)
```

Comment:
Could we consolidate "breaking changes" and "non-breaking changes" into one bullet point with a yes/no

Reply:
It looks weird in .md but github renders it as a checklist so it makes sense to have them separate. Just check the one that applies.

Example:

# Type of change

Please delete options that are not relevant.

- [ ] Non-breaking changes
- [ ] Breaking change (fix, feature or refactoring that changes existing functionality)